# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/performance/browser_preferences_usage.js
# Commit: 2cf5cad90099
# Full Hash: 2cf5cad900992236d596bfb329099742ff15f689
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2024-06-20 04:08:16
# Regressor Bug: 1728331
# File Overlap Count: 1
# Description:
#   Bug 1728331 - Part 4: Make ContentParent KeepAlives explicit with RAII references, r=smaug,dom-worker-reviewers,asuth
#   
#   This is a fairly significant patch, however it would be difficult to break it
#   down into smaller patches:
#   
# ==============================================================================

diff -r d920c2d72d00 -r 2cf5cad90099 browser/base/content/test/performance/browser_preferences_usage.js
--- a/browser/base/content/test/performance/browser_preferences_usage.js	Wed Jun 19 20:14:49 2024 +0000
+++ b/browser/base/content/test/performance/browser_preferences_usage.js	Wed Jun 19 20:14:50 2024 +0000
@@ -186,8 +186,8 @@
     // This pref is only accessed in automation to speed up tests.
     knownProblematicPrefs["dom.ipc.keepProcessesAlive.webIsolated.perOrigin"] =
       {
-        min: 100,
-        max: 102,
+        min: 50,
+        max: 51,
       };
     if (AppConstants.platform == "linux") {
       // The following sandbox pref is covered by